Marlin/src/lcd/extui/lib/mks_ui/tft_lvgl_configuration.cpp

```diff
@@ -79,7 +83,7 @@ extern uint8_t sel_id;
 
 extern bool flash_preview_begin, default_preview_flg, gcode_preview_over;
 
-uint8_t bmp_public_buf[17 * 1024];
+uint8_t bmp_public_buf[14 * 1024];
```

Comment:
How do we guarantee that the largest JPG will be under 14 kilobytes? Or 17?

Although we have eschewed the use of `new`, `delete`, `malloc()`, and `free()` in the past, they are okay for short-lived buffers. So, we can allocate file loading buffers as-needed, as long as they are freed as soon as they are used, to prevent heap fragmentation.

Reply:
We don't need handle it. We just tell to lvgl what is the size of this buffer. And it will handle it, and flush the data to the TFT as it need.
The idea of this buffer is: we should use the max we can, that will not interfere with our tasks.

But it can be smaller.